Lib/test/test_math.py

```diff
@@ -797,6 +808,8 @@ def testHypot(self):
         # Test allowable types (those with __float__)
         self.assertEqual(hypot(12.0, 5.0), 13.0)
         self.assertEqual(hypot(12, 5), 13)
+        self.assertEqual(hypot(1, -1), 1.4142135623730951)
+        self.assertEqual(hypot(1, FloatLike(-1.)), 1.4142135623730951)
```

Comment:
`math.sqrt(2)`

But it is better to test with exactly representable results, integers or fractions with power 2 denominator..

Reply:
I doubt it's possible here: one argument should be ``-1``.